tests/release/const.go

```diff
@@ -18,6 +17,35 @@ const (
 	environment                         = "test-environment"
 	releaseStrategyServiceAccount       = "pipeline"
 
+	sourceReleaseLinkName                string = "source-release-link"
+	targetReleaseLinkName                string = "target-release-link"
+	releasePipelineNameDefault           string = "bundle-release-pipelinerun"
+	applicationNameDefault               string = "appstudio"
+	componentName                        string = "dc-metro-map"
+	buildPipelineBundleDefault           string = "quay.io/redhat-appstudio/hacbs-templates-bundle:latest"
+	buildPipelineBundleDefaultName       string = "build-pipelines-defaults"
+	releaseStrategyPolicyDefault         string = "mvp-policy"
+	releaseStrategyServiceAccountDefault string = "release-service-account"
+	sourceReleasePlanName                string = "source-releaseplan"
+	targetReleasePlanAdmissionName       string = "demo"
+	releasePvcName                       string = "release-pvc"
+	releaseStrategyDefaultName           string = "mvp-strategy"
+	releaseEnvironment                   string = "production" //"sre-production"
+	redhatAppstudioUserSecret            string = "hacbs-release-tests-token"
+	hacbsReleaseTestsTokenSecret         string = "redhat-appstudio-registry-pull-secret"
+	publicSecretNameAuth                 string = "cosign-public-key"
+	gitSourceComponentUrl                string = "https://github.com/sbose78/dc-metro-map"
+	sourceKeyName                        string = "release-e2e+release_e2e"
+	destinationKeyName                   string = "hacbs-release-tests+m5_robot_account"
+	containerImageUrl                    string = "quay.io/hacbs-release-tests/dcmetromap:latest"
+	releasePipelineBundleDefault         string = "quay.io/hacbs-release/pipeline-release:main"
+	roleName                             string = "role-release-service-account"
+	roleBindingName                      string = "role-relase-service-account-binding"
+	subjectKind                          string = "ServiceAccount"
+	roleRefApiGroup                      string = "rbac.authorization.k8s.io"
+	roleRefName                          string = "role-m6-service-account"
+	roleRefKind                          string = "Role"
```

Comment:
having a constant for this is pretty weird imo

Reply:
okay, 
so I just passed the string directly in the calling of func it self to save const